### PR TITLE
add mutation example

### PR DESCRIPTION
**Pull request by notrab**
# Description

Includes a `.vscode/settings.json` file for better DX those using Code when running the example + mutation example.

CleanShot 2022-11-25 at 13 26 44

# Type of change

- [ ] 💔 Breaking
- [x] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [ ] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

File: examples/fresh/utils/grafbase.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    method: 'POST',
+    headers: {
+      'content-type': 'application/json',
+      'x-api-key': Deno.env.get('GRAFBASE_API_KEY') as string
+    },
+    body: JSON.stringify({
+      query,
+      variables
+    })
+  })
```
Comment (vitalikda) [inline examples/fresh/utils/grafbase.ts:19]: Lovely 👍

File: examples/fresh/README.md [hunk 2]
```diff
@@ -14,12 +14,14 @@ This example uses the next-gen web framework Fresh &mdash; [learn more](https://
 4. Run `cp .env.example .env` to copy the example `.env.example` file to `.env`
 5. Open `.env` in your code editor and provide your Grafbase API endpoint and API key. Example should be fine for `npx grafbase dev`!
 6. Run `npx grafbase@latest dev` in your terminal
-7. Populate the backend with some `Post` entries using a GraphQL mutation:
+7. Populate the backend with some `Message` entries using a GraphQL mutation:
```
Comment (vitalikda) [inline examples/fresh/README.md:17]: This step seems unnecessary now

**Review by vitalikda — APPROVED** [review]
Great example 👏

File: examples/fresh/README.md (hunk 2 above)
Comment (notrab) [reply examples/fresh/README.md:17]: Valid point 👍🏻

**Pull request merged**